templates/audit_rules.erb

```diff
@@ -0,0 +1,18 @@
+## First rule - delete all
+-D
+
+## Increase the buffers to survive stress events.
+## Make this bigger for busy systems
+-b <%= @audit_buffer_bytes %>
+
+## This determine how long to wait in burst of events
+--backlog_wait_time <%= @audit_backlog_wait_time %>
+
+## Set failure mode to syslog
+-f 1
+
```

Comment:
As I commented in the previous comment, these rules should be defined in `params_agent.pp`.

I changed it in 5972eeef87dc906a072f41ff1714858d694c8498

Thanks.